Test/comp/ForallNewSyntax.dfy

```diff
@@ -0,0 +1,356 @@
+// RUN: %dafny /quantifierSyntax:4 /compile:3 /spillTargetCode:2 /compileTarget:cs "%s" > "%t"
```

Comment:
Could you verify once, and then use `/compile:4` here as well?

Reply:
Good call, this is just what the original test file was doing.